opm/simulators/linalg/FlowLinearSolverParameters.hpp

```diff
@@ -277,48 +237,48 @@ namespace Opm
     /**
      * \brief Parameters used to configure the CPRPreconditioner.
      */
-    struct CPRParameter
-    {
-        double cpr_relax_;
-        double cpr_solver_tol_;
-        int cpr_ilu_n_;
-        MILU_VARIANT cpr_ilu_milu_;
-        bool cpr_ilu_redblack_;
-        bool cpr_ilu_reorder_sphere_;
-        bool cpr_use_drs_;
-        int cpr_max_ell_iter_;
-        int cpr_ell_solvetype_;
-        bool cpr_use_amg_;
-        bool cpr_use_bicgstab_;
-        int cpr_solver_verbose_;
-        bool cpr_pressure_aggregation_;
-        int cpr_reuse_setup_;
-        CPRParameter() { reset(); }
-
-        void reset()
-        {
-            cpr_solver_tol_           = 1e-2;
-            cpr_ilu_n_                = 0;
-            cpr_ilu_milu_             = MILU_VARIANT::ILU;
-            cpr_ilu_redblack_         = false;
-            cpr_ilu_reorder_sphere_   = true;
-            cpr_max_ell_iter_         = 25;
-            cpr_ell_solvetype_        = 0;
-            cpr_use_drs_              = false;
-            cpr_use_amg_              = true;
-            cpr_use_bicgstab_         = true;
-            cpr_solver_verbose_       = 0;
-            cpr_pressure_aggregation_ = false;
-            cpr_reuse_setup_          = 0;
-        }
-    };
+    // struct CPRParameter
```

Comment:
Do you expect to actually remove this structure,and its associate support code at some point or is this just a temporary workaround?

Reply:
There are some commented-out parts that were left in, initially to see if they could be safely removed. I'll make some cleanup.